src/main/webapp/myprofile.html

```diff
@@ -31,19 +32,43 @@ <h1>User Directory</h1>
       <div class="wrapper">
         <!-- User Details -->
         <div class="details-container">
-            <img src="/images/avatar.jpg"></img>
+            <img class="profilepic" src="/images/avatar.jpg"></img>
             <h2>Name</h2>
             <h3>Department</h3>
             <p>Hello I am interested in __ please feel free to contact me for __</p>
-            <button class="button">Change my profile</p>
+            <button class="button" onclick="togglePopup()">Edit profile</p>
         </div>
         <!-- User's Recommendation Map -->
-        <div id="map"></div>  
+        <div class="map-container">
+          <h2>My favorite places</h2>
+          <div id="map"></div>  
+        </div>
+        <!-- Pop up box to change profile -->
+        <div id="popup-profile">
+          <div class="overlay"></div>
+          <div class="popup-content">
+            <div class="close-button" onclick="togglePopup()">&times;</div>
+            <h2>Edit Profile</h2>
+            <p>Please change your profile by filling up this form and press submit</p>
```

Comment:
nit: This sentence needs rewording: "Please change your profile by filling out this form and press submit"